UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c

```diff
@@ -286,3 +288,39 @@ PlatformBootManagerUnableToBoot (
   return;
 }
 
+
+
+/**
+  Get/update PcdBootManagerMenuFile from GUID HOB which will be assigned in bootloader.
+
+  @retval EFI_SUCCESS       The entry point is executed successfully.
+  @retval other             Some error occurs.
+  
+**/
+EFI_STATUS
+EFIAPI
+PlatformBootManagerLibConstructor (
+  IN EFI_HANDLE        ImageHandle,
+  IN EFI_SYSTEM_TABLE  *SystemTable
+  ) 
+{
+  EFI_STATUS                         Status;
+  UINTN                              size;
+  VOID                               *GuidHob;
+  BOOT_MANAGER_MENU_FILE             *BootManagerMenuFile;
+  Status = EFI_SUCCESS;
+  GuidHob = GetFirstGuidHob (&gUniversalPayloadBootManagerMenuFileGuid);
+  DEBUG((DEBUG_ERROR,"gUniversalPayloadBootManagerMenuFileGuid\n"));
+  //
+  // Find the buffer information and update PCDs
+  //
+  if (GuidHob != NULL) {
+    DEBUG((DEBUG_ERROR,"gUniversalPayloadBootManagerMenuFileGuid2\n"));

```

Comment:
Please check the revision and length, reference
MdeModulePkg/Universal/Acpi/AcpiTableDxe/AcpiTableProtocol.c
line 1748